### DISCUSSION TITLE
Turning off Dark theme on Mac OS

### DISCUSSION THREAD

**Discussion by june-park95** [zim-desktop-wiki/zim-desktop-wiki#2645] [Q&A]
Hi, recently got to know this amazing project and I am very excited to use Zim!

My default system setting for my mac is dark theme and realized that some of the texts are not so visible.

Is there any way to turn it off just for the zim app?

Thank you!

**Comment by june-park95** [comment]
<img width="201" alt="Screenshot 2024-07-25 at 12 46 20 PM" src="https://github.com/user-attachments/assets/dc7f2ee1-686e-43fa-8edd-a48811ac8397">

Also have another question while I am here..!

Why are the pages I created on the index formatted normally, while the page I created using the link is italic and in gray?

Thank you!

**Reply by bruvellu** [reply]
Pages with italic, light gray text are pages that don't exist yet. They show up in the index because they have been linked to from another page. Once you add some content, they'll become a regular page.

**Reply by june-park95** [reply]
Hey! Thank you for the quick answer. In regards to turning off the dark theme, can you please give any guidance?

**Comment by ZimTom** [comment]
I can't find the thread back that I opened with a simialr question, but @dehesselle kindly pointed out the solution to me:
In the [FAQ page](https://www.zim-wiki.org/manual/FAQ.html) under the Question _Can I change the colors used for links, underline etc. ?_ there is a link to a description howw to modify the format settings for the ZIM files, and with a bit of try and error I got it figured out. My problem was that readable text seemed to disappear when the Mac switched to dark mode. I fixed it by modifying the critical formats and it works fine. E.g. I changed underline from the highlighing in yellow to true underlining to keep underlined text visible in dark mode and so it worked for other formats as well, like the dark blue links becoming almost invisible in dark mode.

One mistake I made in the first place was to alter the config.file in the wrong place, because I wasn't familiar enough with the mac structures.

The file to modify is the _styles.conf_ in path _~/Library/Application Support/org.zim-wiki.Zim_  ,i.e. under the user and NOT in the program package.

Because I couldn't find out how to define separate formatting for bright and dark mode  - and I'm not even sure that this is possible - I simply chose formats that are readable in both modes.

Hope this helps a little ...

**Comment by dehesselle** [comment]
Zim 0.76.0 will be released shortly for macOS and it has a GUI setting to switch between light and dark mode. You won't need to modify variables or `zimapp.ini` any longer, and in fact you should undo any modifications you have made to ensure the GUI setting works as intended.

Closing.
